test/extended/builds/s2i_extended_build.go

```diff
@@ -53,22 +52,11 @@ var _ = g.Describe("[builds][Slow] s2i extended build", func() {
 			})
 
 			g.By("running the build")
-			out, err := oc.Run("start-build").Args("--build-loglevel=5", buildConfigName).Output()
+			br, _ := exutil.StartBuildAndWait(oc, buildConfigName, "--build-loglevel=5")
```

Comment:
@php-coder I considered it redundant to the AssertSuccess() which follows the line. i.e. if there were a severe error resulting in an err being returned, AssertSuccess would also fail. As part of AssertSuccess, a significant amount of information about the attempted build is output to the Ginkgolog, so we should have everything we need to diagnose.